components/NewNFTBook.vue

```diff
@@ -557,31 +474,38 @@ const mdEditorPlaceholder = ref({
 const classIdInput = ref(classId || '')
 const nextPriceIndex = ref(1)
 const tableOfContents = ref('')
-const mustClaimToView = ref(true)
 const enableCustomMessagePage = ref(false)
 const hideDownload = ref(false)
 const autoDeliverNftIdInput = ref('')
+const isAllowCustomPrice = ref(true)
+const sessionData = computed(() => {
+  return getUploadFileData()
+})
```

Comment:
A computed property of a store action is unclear in meaning, prefer just calling `getUploadFileData()` everytime

Reply:
[♻️ Compute state data instead of using action to retrieve session data](https://github.com/likecoin/nft-book-press/pull/332/commits/a6e64067e57219fc762252741265b6dc5b2b39ee)